app/Http/Controllers/DinnerformController.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Proto\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\URL;
+use Proto\Models\Dinnerform;
+use Carbon\Carbon;
+
+use Session;
+use Auth;
+use Response;
+
+class DinnerformController extends Controller
+{
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     */
+    public function create()
+    {
+        $dinnerformList = Dinnerform::all()->sortByDesc('end');
+
+        return view('dinnerform.admin', ['dinnerformCurrent' => null, 'dinnerformList' => $dinnerformList]);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function store(Request $request)
+    {
+        $dinnerform = new Dinnerform();
+        $dinnerform->restaurant = $request->restaurant;
+        $dinnerform->description = $request->description;
+        $dinnerform->url = $request->url;
+        $dinnerform->start = strtotime($request->start);
+        $dinnerform->end = strtotime($request->end);
```

Comment:
For stuff like this you should use the Dinnerform::Create() function to pass an array with the data.